modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -986,6 +988,17 @@ def set_derived_appointment_date_fields(appointment)
         appointment[:future] = future?(appointment)
       end
 
+      def log_telehealth_issue(appointment)
+        context = {
+          kind: appointment[:kind],
+          vvsVistaVideoAppt: appointment.dig(:extension, :vvsVistaVideoAppt),
+          modality: appointment[:modality],
+          telehealthUrl: appointment.dig(:telehealth, :url),
```

Comment:
Hmmm I would also include `appointment.dig(:telehealth, :displayLink)`. We should also update the displayLink logic to not require telehealth be non-empty. 

Another nit is that `displayLink` should technically be `display_link`. But we'll need to make sure that change doesn't break something else so.... maybe it's a case of if it ain't broken don't fix it? The argument can also be made that it's kinda broken right now so might as well fix it. So up to you 😄

Reply:
Changed to display_link.... but if we allow telehealth to be empty, how can you store a value nested under it?